tspng/implantation.py

```diff
@@ -0,0 +1,82 @@
+# import statements
+import os
+import json
+
+from PIL import Image
+from PIL.PngImagePlugin import PngInfo
+from tspng import MIME_TYPE
+from typing import Dict
+
+
+def _implant_data(data: str, image: str, mime_type: str = MIME_TYPE, dst: str = None):
```

Comment:
If the `dst` is not going to be used/implemented, then let's leave it out for now and add with a separate PR.